sqlalchemy-stubs/engine/base.pyi

```diff
@@ -97,7 +97,7 @@ class Engine(Connectable, log.Identified):
     def begin(self, close_with_result: bool = ...): ...
     def transaction(self, callable_, *args, **kwargs): ...
     def run_callable(self, callable_, *args, **kwargs): ...
-    def execute(self, statement, *multiparams, **params): ...
+    def execute(self, statement: TextClause, *multiparams, **params): ...
```

Comment:
Can be more than TextClause: https://docs.sqlalchemy.org/en/13/core/connections.html#sqlalchemy.engine.Connection.execute